bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/XMLSequence.java

```diff
@@ -70,6 +70,12 @@ public XMLSequence(List<BXML> children) {
         this.children = children;
     }
 
+    public XMLSequence(BXML child) {
+        this();
```

Comment:
Why do we need `this()` since we are doing `this.children = new ArrayList<>();` after anyway?

Reply:
Will remove extra calls